src/volume.ts

```diff
@@ -1493,7 +1493,8 @@ export class Volume {
     const realLink = this.getResolvedLink(steps);
     if (!realLink) throw createError(ENOENT, 'realpath', filename);
 
-    return strToEncoding(realLink.getPath(), encoding);
+    const path = realLink.getPath();
+    return strToEncoding(path ? path : '/', encoding);
```

Comment:
we can just use `||` here right?

```suggestion
    return strToEncoding(realLink.getPath() || '/', encoding);
```

Reply:
btw fwiw this behavior technically is not correct for Windows, but I think this is still better than the current behaviour so happy to land it

![image](https://user-images.githubusercontent.com/3151613/198850585-e05a3aa1-9949-42fb-91f0-726eb7ced6a1.png)